tests/validate-plugin.sh

```
@@ -136,21 +147,28 @@ echo ""
+# and COMMAND_COUNT would be discarded on exit.
 COMMAND_COUNT=0
-for cmd in "$PLUGIN_ROOT/commands"/*.md; do
-    if [ -f "$cmd" ]; then
-        cmd_name=$(basename "$cmd")
-        if [ "$cmd_name" != "README.md" ]; then
-            # Check for frontmatter
-            if head -1 "$cmd" | grep -q "^---"; then
-                pass "$cmd_name has frontmatter"
+if [ -d "$PLUGIN_ROOT/commands" ]; then
+    while IFS= read -r -d '' cmd; do
+        cmd_rel="${cmd#$PLUGIN_ROOT/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)